InteractionBase/src/handlers/GenericHandler.cpp

```diff
@@ -510,6 +514,33 @@ void GenericHandler::keyPressEvent(Visualization::Item *target, QKeyEvent *event
 
 		scene->scheduleUpdate();
 	}
+	else if (event->key() == Qt::Key_F12)
+	{
+		event->accept();
+		if (SelectionAtCursorItem::isVisible())
+			SelectionAtCursorItem::hide();
+		else
+		{
+			QVector<Model::Node*> nodes;
+			for (auto view : target->scene()->viewItems())
+				nodes.append(new Interaction::ViewSelectionNode(view->name()));
```

Comment:
Is the one and only reason you have created this ViewSelectionNode node, in order to use it in a Grid? If a grid could display items directly, would that have been fine for you?